awx/ui_next/src/screens/ExecutionEnvironment/shared/ExecutionEnvironmentForm.jsx

```diff
@@ -37,8 +47,28 @@ function ExecutionEnvironmentFormFields({ i18n, me, executionEnvironment }) {
     [setFieldValue]
   );
 
+  const [
+    containerOptionsField,
+    containerOptionsMeta,
+    containerOptionsHelpers,
+  ] = useField({
+    name: 'container_options',
+  });
+
+  const containerPullChoices = options?.actions?.POST?.container_options?.choices.map(
+    ([value, label]) => ({ value, label, key: value })
+  );
+
   return (
     <>
+      <FormField
+        id="execution-environment-name"
+        label={i18n._(t`Name`)}
+        name="name"
+        type="text"
+        validate={required(null, i18n)}
+        isRequired
```

Comment:
Not sure if name will be a required field on the API side.

Reply:
I can make it one, and in fact think it's prudent to have it be